src/assets/images/spinner.svg

```diff
@@ -0,0 +1 @@
+<svg height='100px' width='100px'  fill="#000000" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24" version="1.1" x="0px" y="0px"><title>icon/loading-spinner-solid</title><desc>Created with Sketch.</desc><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><path d="M12,23 C5.935,23 1,18.065 1,12 C1,8.086 3.106,4.436 6.498,2.473 L7.5,4.203 C4.725,5.81 3,8.797 3,12 C3,16.963 7.037,21 12,21 C16.963,21 21,16.963 21,12 C21,7.037 16.963,3 12,3 L12,1 C18.065,1 23,5.935 23,12 C23,18.065 18.065,23 12,23" fill="#000000"></path></g></svg>
```

Comment:
added this because `spinner-solid` was not looking nice when used in the way that I was using - this was looking neat.